Tests/GitHubCodespaces.tests.ps1

```
@@ -34,6 +34,59 @@ BeforeAll {
     $mainBranchName = $repo | Get-GitHubRepositoryBranch | Select-Object -ExpandProperty name -First 1
 }
 
+Describe 'GitHubCodespaces\Set-GitHubCodespaceVisibility' {
+    Context 'When setting the visibility of a codespace' {
+
+        It 'sets the visibility successfully too ''disabled'' ' {
+            $setVisibilityParams = @{
+                Force = $true
+                OrganizationName = $script:organizationName
+                Visibility = 'disabled'
+            }
+            Set-GitHubCodespaceVisibility @setVisibilityParams
+        }
+
+        It 'sets the visibility successfully too ''all_members'' ' {
+            $setVisibilityParams = @{
+                Force = $true
+                OrganizationName = $script:organizationName
+                Visibility = 'all_members'
+            }
+            Set-GitHubCodespaceVisibility @setVisibilityParams
+        }
+        It 'sets the visibility successfully too ''selected_members'' ' {
+            $setVisibilityParams = @{
+                Force = $true
+                OrganizationName = $script:organizationName
+                UserName = $script:ownerName
+                Visibility = 'selected_members'
+            }
+            Set-GitHubCodespaceVisibility @setVisibilityParams
+        }
+
+    }
+}
+
+Describe 'GitHubCodespaces\Remove-GitHubCodespaceUser' {
```

Can we run this test successfully multiple times by itself, or does it depend on a previous test to set up the correct state?  In other words, if I remove `octocat` the first time, it shouldn't throw.  If I then try to remove `octocat` a second time, does the API care, or will I get a different response?

---

Both Add and Remove GitHubCodespaceUser will throw HTTP Status 304 Not Modified if called multiple times for the same user(s).  I've added code to suppress this error in a4ecc471a22c575e6e5edd091e9e6746d9038b21.  Let me know if you have an alternate suggestion.